rebuild-awstats/rebuild-awstats.sh

```diff
@@ -223,6 +223,17 @@ merge_logs() {
     $AWSTATS_TOOLS_D/logresolvemerge.pl "$@" 
 }
 
+check_log_sort(){
+    local log_file=$1
+
+    LC_TIME=C sort -s -t ' ' -k 4.9n -k 4.5M -k 4.2n -C "$log_file" && return
+
+    echo "WARNING: Сontents of log files not sorted, will running sorting logs, this operation may take a long time"
```

Comment:
1. I don't think this will give the user any additional information
3. statistics_log_processor for logs concationation used logresolvemerge, so I don't think this situation is common

Reply:
1. OK as this is a temporary file. Will it be at least clear which domain this relates to?
2. OK.
3. `logresolvemerge` is used only for logs which serve as input for web statistics. These are not the same logs as the ones rotated by `logrotate` and processed by this script.

Reply:
1. Yes, domain name can see in output 
`=== ip-10-69-45-248.eu-west-1.compute.internal
  Saving: webstat* --> webstat*.95
  Log files are not sorted by timestamp. They will be sorted, but this operation is resource-intensive and may take a long time.
  Logs begin on: 
    access_log* (new/old/corrupted): 0/0/0
    access_ssl_log* (new/old/corrupted): 738/0/0
  Rebuilding static pages: 
2022-12 Successfully rebuild all html files.
=== festive-panini.10-69-45-248.qa.plesk.tech
  Saving: webstat* --> webstat*.2
  Logs begin on: 
    access_log* (new/old/corrupted): 0/0/0
    access_ssl_log* (new/old/corrupted): 0/0/0
  Rebuilding static pages: 
2022-12 Successfully rebuild all html files.
=== objective-hopper.10-69-45-248.qa.plesk.tech
  Saving: webstat* --> webstat*.2
  Logs begin on: 
    access_log* (new/old/corrupted): 0/0/0
    access_ssl_log* (new/old/corrupted): 0/0/0
  Rebuilding static pages: 
2022-12 Successfully rebuild all html files.`
3. OK, so i remove WARNING